src/CountryListService.php

```diff
@@ -4,18 +4,11 @@
 
 class CountryListService
 {
-    /**
-     * @return array
-     */
     public function blacklistedCountries(): array
     {
-        return ['AL','BY','BA','BI','CF','HR','HU','CU','KP','CD','IR','IQ','QZ','LA','LB','LY','MK','ME','NI','RS','SO','SS','SD','SY','UA','VE','YE','ZW','BS','BB','BJ','BF','KH','KY','TD','CO','KM','CG','DO','GQ','ER','GN','GW','HT','LS','LR','MG','MM','PA','UG','PR'];
```

Comment:
We need to review this and make sure that Galileo (and other non-applicable vendors) can not be sent to these countries; IF our services rely on determining the eligibility of a product based on this list. I am not 💯 sure that our cart / checkout functionality even checks against this list, it may have it's own list based on product configuration?? Some discovery needs to be done here first.

We've started talking with Kim in Devops.